package.json

```diff
@@ -36,6 +36,7 @@
     "node": ">=4"
   },
   "dependencies": {
+    "@gimenete/type-writer": "^0.1.3",
```

Comment:
Should this be a production dependency, or should it be in `devDependencies`?

Reply:
@gimenete I assume it’s not needed in production?

Reply:
@gr2m no, it’s not needed in production